smarts/sstudio/types/scenario.py

```diff
@@ -34,15 +39,98 @@
 from smarts.sstudio.types.zone import RoadSurfacePatch
 
 
+class MetadataFields(IntEnum):
+    actor_of_interest_color = enum.auto()
+    actor_of_interest_re_filter = enum.auto()
+    scenario_difficulty = enum.auto()
+    scenario_length = enum.auto()
+
+
 @dataclass(frozen=True)
 class ScenarioMetadata:
-    """Scenario data that does not have influence on simulation."""
+    """Deprecated. Scenario data that does not have influence on simulation."""
```

Comment:
Is there a reason we didn't just change this class? I like the original name better.

Reply:
Right, I had changed this already by the time you commented. I forgot to push the updates here.